packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -33,10 +33,20 @@ import 'inherited_notifier.dart';
 ///
 /// Managing a [FocusNode] means managing its lifecycle, listening for changes
 /// in focus, and re-parenting it when needed to keep the focus hierarchy in
-/// sync with the widget hierarchy. See [FocusNode] for more information about
-/// the details of what node management entails if not using a [Focus] widget.
+/// sync with the widget hierarchy. This widget does all of those things for
+/// you. See [FocusNode] for more information about the details of what node
+/// management entails if you are not using a [Focus] widget and you need to do
+/// it yourself.
 ///
-/// To collect a sub-tree of nodes into a group, use a [FocusScope].
+/// To collect a sub-tree of nodes into an exclusive group that restricts focus
+/// traversal to the group, use a [FocusScope]. To collect a sub-tree of nodes
+/// into a group that has a specific order to its traversal but allows the
+/// traversal to escape the group, use a [FocusTraversalGroup].
+///
+/// To move the focus, use methods on [FocusNode] by getting the [FocusNode]
+/// through the [of] method. For instance, to move the focus to the next node in
+/// the focus traversal order, call `Focus.of(context).nextFocus()`. To unfocus
+/// a widget, call `Focus.of(context).unfocus()`.
```

Comment:
Is this a spot where we can clarify that unfocus will focus on something else instead of nothing?

Reply:
Well, yes, but at the moment (until https://github.com/flutter/flutter/pull/50831 lands), that's exactly where it will go.  Well, technically it focuses the `FocusManager.instance.rootScope`, but that's basically nowhere, since once that has focus, the focus traversal can't find a new node to focus.

Reply:
I'll update this comment once the other PR lands (added a TODO).

Reply:
Sgtm!